.github/workflows/mkdocs-delete-version.yml

```diff
@@ -0,0 +1,38 @@
+name: Delete Documentation Version
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version string to be deleted (e.g."v0.0.1")'
+        required: true
+jobs:
+  build:
+    name: Delete docs Version
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install git+https://${{ secrets.GH_TOKEN }}@github.com/lensapp/mkdocs-material-insiders.git
+          pip install mike
+          
+      - name: Checkout Release from lens
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
@msa0311 - shouldn't we check out the tag here rather than master?

Reply:
As far as I understood how actions/checkout@v2 works it uses github.ref as default which should always be the correct tag / branch. 

https://github.com/actions/checkout

```
   # The branch, tag or SHA to checkout. When checking out the repository that
    # triggered a workflow, this defaults to the reference or SHA for that event.
    # Otherwise, uses the default branch.
    ref: ''
```

Reply:
Yes but here the branch is not triggering the action as its manual - the user is is supplying the tag and as we do not specify anything in the check I assume it will default to master/main?

Reply:
Can review later if needed